pcsx2/Patch.cpp

```diff
@@ -386,6 +420,39 @@ void Patch::EnumeratePnachFiles(const std::string_view& serial, u32 crc, bool ch
 		f(std::move(zip_filename), std::move(pnach_data.value()));
 }
 
+bool Patch::PatchStringHasUnlabelledPatch(std::string pnach_data)
+{
+	std::istringstream ss(pnach_data);
```

Comment:
Is `istringstream` used elsewhere to load pnachs? If it's not, you may need to make it loaded the same way as in other places.